master/internal/core.go

```diff
@@ -429,14 +435,11 @@ func (m *Master) startServers(ctx context.Context, cert *tls.Certificate) error
 		}()
 	}
 	start("gRPC server", func() error {
```

Comment:
any reason we don't start the cmux mux first?

Reply:
The order doesn't really matter, since they're all in goroutines anyway, right?

Reply:
Indeed